Webmin/input/checks/accounts_passwd_cmd.xml

```diff
@@ -0,0 +1,33 @@
+<def-group>
+  <definition class="compliance" id="accounts_passwd_cmd" version="1">
+    <metadata>
+      <title>Use passwd for changing passwords.</title>
+      <affected family="unix">
+        <product>Webmin</product>
```

Comment:
This will be a bit tedious. I have typically been testing with the tip. But your right. Won't be done overnight, but I will look into validating when each needed feature is introduced and reference said version in each rule.